### PR TITLE
Update config.yml

### LINKED ISSUE #1706: FTO: Fix typos and update config.yml
Hi, this is a [first-timers-only issue](https://code.publiclab.org/#r=all), and this means we've worked to make it more legible to folks who either **haven't contributed to our codebase before, or even folks who haven't contributed to open source before**.

If that's you, we're interested in helping you take the first step and can answer questions and help you out as you do. Note that we're especially interested in contributions from people from groups underrepresented in free and open source software!

We know that the process of creating a pull request is the biggest barrier for new contributors. This issue is for you 💝

If you have contributed before, **consider leaving this one for someone new** and looking through our general [help wanted](https://github.com/publiclab/mapknitter/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) issues. Thanks!

### 🤔 What you will need to know.

Nothing. This issue is meant to welcome you to Open Source :) We are happy to walk you through the process.

### 📋 Step by Step

- [ ] 🙋 **Claim this issue**: Claim the issue by commenting. If someone else has claimed it, ask if they've opened a pull request already and if they're stuck -- maybe you can help them solve a problem or move it along!
<details>
  <summary>💡 <i>Learn how to claim</i> 🙋</summary>
  <h3>Claiming an issue</h3>
  <p>Unless the issue is marked as reserved for someone, you can just say "I'd like to try this!" and then you've claimed it - no need to wait for someone to assign it to you. Just be sure you link your pull request (PR) to this issue so we can see where your solution is.</p>
  <p>And open one early if possible - even before you've completed it with additional commits - and others can help you figure out any issues you may face.</p>
</details>

- [ ] 📝 **Update** the file [config.yml](https://github.com/publiclab/mapknitter/blob/main/.github/config.yml) in the `mapknitter` repository (press the little pen Icon) and edit the line as shown below.

[See this page](https://code.publiclab.org/#r=all) for some help in taking your first steps!

Below is a "diff" showing in red (and a `-`) which lines to remove, and in green (and a `+`) which lines to add:

```diff
@@ -2,25 +2,29 @@
 # Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
 
-# Comment to be posted to on first time issues
+# Comment to be posted on first time issues
 newIssueWelcomeComment: |
   Thanks for opening your first issue here! Please follow the issue template to help us help you 👍🎉😄
   If you have screenshots to share demonstrating the issue, that's really helpful! 📸 You can [make a gif](https://www.cockos.com/licecap/) too!
+  Do join our [Gitter channel](https://gitter.im/publiclab/publiclab) for some brainstorming discussions.
 # Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
 
-# Comment to be posted to on PRs from first time contributors in your repository
+# Comment to be posted on PRs from first time contributors in your repository
 newPRWelcomeComment: |
  Thanks for opening this pull request! `Dangerbot` will test out your code and reply in a bit with some pointers and requests.
  There may be some errors, **but don't worry!** We're here to help! 👍🎉😄
+ <br />
+ One thing that can help to get started is to make sure you've included a link back to the original issue you're solving, in the format `fixes #0000` (for example). And to make sure the PR title describes what you're trying to do! (often it can be the same as the issue title) Thanks! 🙌 
+ <br />
+ Then, you can [say hello in our chatroom](https://gitter.im/publiclab/publiclab) & share a link to this PR to get a review! 👋 ✅
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 
-# Comment to be posted to on pull requests merged by a first time user
+# Comment to be posted on pull requests merged by a first time user
 firstPRMergeComment: |
  Congrats on merging your first pull request! 🙌🎉⚡️ 
- Your code will likely be published to https://mapknitter.org in the next few days. 
- In the meantime, can you tell us your Twitter handle so we can thank you properly?
+ Your code will likely be published to https://mapknitter.org in the next few days.
  Now that you've completed this, you can help someone else take their first step!
- Reach out to someone else working on theirs on [Public Lab's code welcome page](https://code.publiclab.org#r=all). Thanks!
+ Reach out to someone else working on theirs on [Public Lab's code welcome page](https://code.publiclab.org#r=all) (where you'll now be featured as a recent contributor!). Thanks!
  <details>
  <summary><b>Help others take their first step</b></summary>
  <p>Now that you've merged your first pull request, you're the perfect person to help someone else out with this challenging first step. 🙌 </p>
  <p><a href="https://code.publiclab.org#r=all">https://code.publiclab.org</a></p>
  <p>Try looking at this list of `first-timers-only` issues, and see if someone else is waiting for feedback, or even stuck! 😕 </p>
  <p>People often get stuck at the same steps, so you might be able to help someone get unstuck, or help lead them to some documentation that'd help. Reach out and be encouraging and friendly! 😄 🎉 </p>
  <p>Read about <a href="https://github.com/publiclab/plots2/issues/3458">how to help support another newcomer here</a>, or find <a href="https://github.com/publiclab/plots2/labels/support">other ways to offer mutual support here</a>.</p>
  </details>
+ <br />
+ You can also join our [Gitter channel](https://gitter.im/publiclab/publiclab) to connect with other contributors :inbox_tray:
 
 # It is recommended to include as many gifs and emojis as possible
```

- [ ] 💾 **Commit** your changes

- [ ] 🔀 **Start a Pull Request**. There are two ways how you can start a pull request:

1. If you are familiar with the terminal or would like to learn it, [here is a great tutorial](https://app.egghead.io/playlists/how-to-contribute-to-an-open-source-project-on-github) on how to send a pull request using the terminal.

2. You can also [edit files directly in your browser](https://help.github.com/articles/editing-files-in-your-repository/) and open a pull request from there.

- [ ] 🏁 **Done** Ask in comments for a review :)

### Please keep us updated

💬⏰ - We encourage contributors to be respectful to the community and provide an update **within a week** of claiming a first-timers-only issue. We're happy to keep it assigned to you as long as you need if you update us with a request for more time or help, but if we don't see any activity a week after you claim it we may reassign it to give someone else a chance. Thank you in advance!

If this happens to you, don't sweat it! Grab another open issue.

### Is someone else already working on this?

🔗- We encourage contributors to link to the original issue in their pull request so all users can easily see if someone's already started on it.

👥- **If someone seems stuck, offer them some help!** Otherwise, [take a look at some other issues you can help with](https://code.publiclab.org/#r=all). Thanks!

### 🤔❓ Questions?

Leave a comment below, and [reach out in our chatroom](https://publiclab.org/chat)!

### PR DESCRIPTION
**Pull request by TenTraicion**
Fixes #1706  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

[//]: # (To mark checkbox write 'x' within the square brackets)

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [x] ask `@publiclab/mapknitter-reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by TenTraicion** [comment]
asking for help here, kindly review this.
@publiclab/mapknitter-reviewers

File: .github/config.yml [hunk 1]
```diff
@@ -29,5 +33,7 @@ firstPRMergeComment: |
  <p>People often get stuck at the same steps, so you might be able to help someone get unstuck, or help lead them to some documentation that'd help. Reach out and be encouraging and friendly! 😄 🎉 </p>
  <p>Read about <a href="https://github.com/publiclab/plots2/issues/3458">how to help support another newcomer here</a>, or find <a href="https://github.com/publiclab/plots2/labels/support">other ways to offer mutual support here</a>.</p>
  </details>
+ <br />
```
Comment (TildaDares) [inline .github/config.yml:36]: Great job @TenTraicion!! Can you remove line 36? Thanks!!

Comment (TenTraicion) [reply .github/config.yml]: Removed the line. Kindly check @TildaDares and Thanks

**Review by TildaDares — APPROVED** [review]
Thank you for working on this @TenTraicion!!

**Comment by TildaDares** [comment]
Congratulations @TenTraicion 🎉

**Comment by TenTraicion** [comment]
> Congratulations @TenTraicion 🎉

Thanks a lot for the help..! @TildaDares

**Pull request merged**